Networking/Networking/Remote/StripeRemote.swift

```diff
@@ -20,16 +21,14 @@ public class StripeRemote: Remote {
 
     /// TODO loadConnectionToken(for siteID: Int64,...)
 
-    /// Captures a payment for an order. See https://stripe.com/docs/terminal/payments#capture-payment
+    /// Captures a payment for an order and returns a publisher of the result. See https://stripe.com/docs/terminal/payments#capture-payment
     /// - Parameters:
     ///   - siteID: Site for which we'll capture the payment.
     ///   - orderID: Order for which we are capturing the payment.
     ///   - paymentIntentID: Stripe Payment Intent ID created using the Terminal SDK.
-    ///   - completion: Closure to be run on completion.
     public func captureOrderPayment(for siteID: Int64,
                                orderID: Int64,
-                               paymentIntentID: String,
-                               completion: @escaping (Result<RemotePaymentIntent, Error>) -> Void) {
+                               paymentIntentID: String) -> AnyPublisher<Result<RemotePaymentIntent, Error>, Never> {
```

Comment:
Not a blocker, but I am wondering whether we could simplify the return publisher by returning a failable one without a result object :)
`AnyPublisher<RemotePaymentIntent, Error>`

Reply:
Good question! I personally prefer keeping the networking error in the result value, like how [Alamofire implements their publisher for Combine](https://github.com/Alamofire/Alamofire/blob/f7e169cd9c5e6b1a9d7403fc526a635fad0a787a/Source/Combine.swift#L33-L37). This way, if there are any other steps in the downstream chain we can determine how to handle the error scenario instead of just skipping them when an error occurs. What do you think? If you have other thoughts, we can p2 this for iOS discussions 🙂